src/pages/Customer.js

```diff
@@ -143,28 +147,46 @@ class CustomerPage extends Component {
   }
 
   handleDocumentNumber = (event) => {
-    const { documentLabel } = this.state
+    const { key } = event
     let document = 'CPF'
-    let value = path(['target', 'value'], event)
+    let value = clean(path(['target', 'value'], event))
+    const documentLength = value.length
+    const hasDocumentReachedLengthLimit = documentLength >= 11
+
+    const pressedNumber = item => item.match(/[0-9]/)
+    const pressedBackspace = item => item === 'Backspace'
+    const pressedDelete = item => item === 'Delete'
+
+    const shouldUpdate = anyPass([
+      pressedNumber,
+      pressedBackspace,
+      pressedDelete,
+    ])(key)
+
+    if (!shouldUpdate) {
```

Comment:
@allangrds acho que fica mais simples se você usar o `shouldUpdate` como função, exemplo:

```js
const shouldUpdate = anyPass([
  pressedNumber,
  pressedBackspace,
  pressedDelete,
])
```
Assim você pode usar o `shouldUpdate` dessa forma:

```suggestion
    if (!shouldUpdate(key)) {
```